src/Apps/Settings/Routes/Insights/Components/CareerHighlights/InsightsCareerHighlightPromoCard.tsx

```diff
@@ -19,15 +31,11 @@ const { src: mSrc, srcSet: mSrcSet } = resized(
   }
 )
 
-export const InsightsCareerHighlightPromoCard: React.FC = () => {
+export const InsightsCareerHighlightPromoCard: React.FC<{
+  onClick?(): void
+}> = ({ onClick }) => {
   return (
-    <Flex
-      width={[205, 313]}
-      height={[135, 178]}
-      background="white"
-      border="1px solid"
-      borderColor="black10"
-    >
+    <CardWrapper onClick={onClick} width={[205, 313]} height={[135, 178]}>
```

Comment:
I see that these were already there but: These hardcoded widths and heights are bad news — leads to brittle UI that _will_ break. You can use `ResponsiveBox` to create fluid width items that are constrained to a specific aspect ratio.

Reply:
Thanks, you basically never want something with a fixed dimension like this.

Reply:
I tried playing with it but it's always sort of squashed when the screen gets smaller 
<img width="841" alt="image" src="https://user-images.githubusercontent.com/20655703/199795132-a275f9da-80f4-494c-924a-aeda9f49d0eb.png">
maybe I'm doing something wrong, but is it because the components are in a `Shelf`?

Reply:
Oh yes! Sorry anything in a Shelf needs some kind of fixed width! Sorry didn't realize what we had here. This is in fact one of the few cases you would want a fixed set of dimensions 😅